smtk/extension/qt/qtUIManager.cxx

```diff
@@ -219,18 +220,20 @@ qtUIManager::~qtUIManager()
 void qtUIManager::initializeUI(QWidget* pWidget, bool useInternalFileBrowser)
 {
   m_useInternalFileBrowser = useInternalFileBrowser;
-  smtk::common::ViewPtr v = this->m_AttSystem.findView(this->m_topViewName);
-  if(!v)
-    {
-    return;
-    }
+  this->m_parentWidget = pWidget;
   if(this->m_topView)
     {
     delete this->m_topView;
+    this->m_topView = NULL;
+    }
+  
+  if(!this->m_smtkView)
+    {
+    return;
```

Comment:
Some message here, such as top level view is not set, will be helpful for debugging purpose